client/src/components/Main/EventsSection/Events/index.js

```diff
@@ -69,6 +69,7 @@ const Events = ({
             </Grid>
           )}
           {visibleEvents.map((event, index) => {
+            let color = event.color ?? randColors[Math.floor(Math.random() * randColors.length)];
```

Comment:
I don't think that we need the randColors part, the backend should be setting the default color to the theme color.